tests/features/lang/language-selector.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Select language 
+
+  Scenario: Change language
```

Comment:
To fix this when Nuxt routing is used, replace with:
```cucumber
    When I visit `/de/search`
    And I click the `language selector`
    And I see a `Svenska language option` in the `language selector`
    And I click the `Svenska language option`
    And I wait 2 seconds
    Then I should be on `/sv/search`
    And I see the text "Vad letar du efter?" in the `search box` placeholder
```

The issue seems to be when `detectBrowserLanguage` is enabled in nuxt-i18n, and the first URL visited in the feature test does not include the language code, but only in headless mode. (I do not know why.)